raiden-ts/src/channels/utils.ts

```diff
@@ -91,6 +91,12 @@ export function assertTx(
   error: ErrorCodes,
   { log }: Pick<RaidenEpicDeps, 'log'>,
 ): OperatorFunction<ContractTransaction, Hash> {
+  /**
+   * Operator to check for tx
+   *
+   * @param tx - pending contract tx
+   * @returns Observable of txHash
+   */
```

Comment:
Call me crazy but this just seems wrong 😅

Reply:
Only way I could make `eslint` happy. Seems they changed something recently regarding those functions which return functions. It's ugly, but makes sense to have docstrings for outer and inner functions.

Reply:
No it doesn't make any sense at all 😅